src/main/java/org/isf/serviceprinting/manager/PrintManager.java

```diff
@@ -1,24 +1,3 @@
-/*
```

Comment:
Come on....what are you doing removing licensing information?

Reply:
Hi, Sorry for the inconvenience. I have added the the license information into PrintManager Class. Can you please review the changes I have made and if possible accept the pull request. The refactoring was part of my University assignment and I need to submit it today.